src/main/java/com/limechain/network/protocol/transaction/state/Validity.java

```diff
@@ -0,0 +1,24 @@
+package com.limechain.network.protocol.transaction.state;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigInteger;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Validity {
+    private BigInteger priority;
+    private byte[][] requires;
+    private byte[][] provides;
+    private BigInteger longevity;
+    private boolean propagate;
+
+    public Validity(BigInteger priority){
+        this.priority = priority;
+    }
+}
```

Comment:
Add a comment explaining what this class represents and how the priority works. If I have priority 1 and someone else has 18, am I with a higher priority or not? Also how are the negative cases handled? Would be nice to have a test for that case as well.